lib/index.js.flow

```diff
@@ -27,7 +27,8 @@ type Options<T: Element> = {|
   constructor?: Class<T>,
   initialize?: InitializerCallback<T>,
   add?: AddCallback<T>,
-  remove?: RemoveCallback<T>
+  remove?: RemoveCallback<T>,
+  subscribe?: SubscribeCallback<T>
```

Comment:
An alternative might be to reuse `initialize` and support `InitializerCallbacks | Subscription` as a valid return object.